testing/ember-app/app/components/util/function.gts

```diff
@@ -22,8 +22,8 @@ export default class GlintTest extends Component {
 
 
   <template>
-    isLoading: {{if this.aMap.isLoading 'true' 'false'}}
-    isError: {{if this.aMap.isError 'true' 'false'}}
+    isLoading: {{if this.aMap.isPending 'true' 'false'}}
+    isError: {{if this.aMap.isRejected 'true' 'false'}}
```

Comment:
Should we rename the left part too?

Reply:
they are aliases now -- more work has happened to this code since merging :sweat_smile: